database/postgresql/postgresql.go

```diff
@@ -119,12 +120,33 @@ VALUES ($1, $2, $3, $4, $5, $6) ON CONFLICT DO NOTHING`
 	return err
 }
 
+// CreateTxPartition implements database.Database
+func (db *Database) CreatePartition(table string, height int64) (int64, error) {
+
+	partitionId := height / int64(config.Cfg.Database.PartitionSize)
```

Comment:
should we check here if the partition_size is not set in the config, return error ?

Reply:
I think it's better to have a default partition size instead @huichiaotsou. This operation will indeed error if `PatitionSize` is set to `0` because it will try to divide by `0` which will panic. We either: 
1. have a default `PartitionSize` 
2. allow for `0` value, meaning no partition will ever be created